include/tudat/simulation/estimation_setup/orbitDeterminationManager.h

```diff
@@ -1031,17 +1032,19 @@ class OrbitDeterminationManager
         {
             variationalEquationsSolver_ =
                     simulation_setup::createVariationalEquationsSolver< ObservationScalarType, TimeType >(
-                        bodies, propagatorSettings, parametersToEstimate_, propagateOnCreation );
+                        bodies, propagatorSettings, parametersToEstimate_, propagateOnCreation, true );
         }
 
         if( integrateAndEstimateOrbit_ )
         {
             stateTransitionAndSensitivityMatrixInterface_ = variationalEquationsSolver_->getStateTransitionMatrixInterface( );
+            dependentVariablesInterface_ = variationalEquationsSolver_->getDynamicsSimulatorBase( )->getDependentVariablesInterface( );
```

Comment:
This fails for the hybrid arc, where the getDynamicsSimTulatorBase function is undefined. I've made a temporary fix, so that the dependent variable interface is not retrieved when the propagation is hybrid arc. By putting the dependent variable interface into the SimulationResults, this can be re-introduced quite easily.